src/main/kotlin/apply/ui/admin/evaluation/EvaluationForm.kt

```diff
@@ -0,0 +1,67 @@
+package apply.ui.admin.evaluation
+
+import apply.application.EvaluationRequest
+import apply.domain.evaluation.Evaluation
+import apply.domain.recruitment.Recruitment
+import com.vaadin.flow.component.button.Button
+import com.vaadin.flow.component.textfield.TextArea
+import com.vaadin.flow.component.textfield.TextField
+import support.views.BindingFormLayout
+import support.views.createErrorSmallButton
+import support.views.createItemSelect
+import support.views.createPrimarySmallButton
+
+class EvaluationForm() : BindingFormLayout<EvaluationRequest>(EvaluationRequest::class) {
+    private val title: TextField = TextField("평가명")
+    private val description: TextArea = TextArea("설명")
+    private val recruitment = createItemSelect<Recruitment>("모집").apply {
+        setItemLabelGenerator(Recruitment::title)
+        isEmptySelectionAllowed = false
+    }
+    private val beforeEvaluation = createItemSelect<Evaluation>("이전 평가").apply {
+        setItemLabelGenerator(Evaluation::title)
+    }
+    private val evaluationItems: MutableList<EvaluationItemForm> = mutableListOf()
+
+    constructor(recruitments: List<Recruitment>, listener: (id: Long) -> List<Evaluation>) : this() {
+        recruitment.setItems(recruitments)
+        recruitment.addValueChangeListener {
+            beforeEvaluation.setItems(listener.invoke(it.value.id))
+        }
+        add(title, recruitment, beforeEvaluation, description)
+        setColspan(title, 2)
+        setColspan(recruitment, 2)
+        setColspan(beforeEvaluation, 2)
+        setColspan(description, 2)
```

Comment:
모든 컴포넌트의 스타일이 동일한 경우 한 줄로 바꾸는 것은 어떨까요?

```suggestion
        setResponsiveSteps(ResponsiveStep("0", 1))
```

Reply:
이런게 있었군요 👍 
수정했습니다.